docs/yul.rst

```diff
@@ -39,7 +39,7 @@ and ``mod`` are available either natively or as functions and computes exponenti
 .. code::
 
     {
-        function power(base:u256, exponent:u256) -> result:u256
+        function power(base:u256, exponent:u256) ‑> result:u256
```

Comment:
Let's not do it here - it will not work for the compiler.

Reply:
Ah yes, sorry!